aws_lambda_builders/runtime.py

```diff
@@ -0,0 +1,66 @@
+"""
+Supported Runtimes and their validations.
+"""
+
+import os
+import subprocess
+from enum import Enum
+
+from aws_lambda_builders.exceptions import MisMatchRuntimeError
+
+
+def validate_python_cmd(required_language, required_runtime_version):
+    major, minor = required_runtime_version.replace(required_language, "").split('.')
+    cmd = [
+        "python",
+        "-c",
+        "import sys; "
+        "assert sys.version_info.major == {major} "
+        "and sys.version_info.minor == {minor}".format(
+            major=major,
+            minor=minor)]
+    return cmd
+
+
+_RUNTIME_VERSION_RESOLVER = {
+    "python": validate_python_cmd
+}
+
+
+class Runtime(Enum):
```

Comment:
wait, this needs to work in Py2.7. Enum isn't available there and I am not interested in taking a new dependency on Enum because it doesn't offer a whole lot of functionality in this case ;) 

Can you refactor this to remove Enum?

Reply:
Sure removed the enum, its a simple class now.